src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -6181,6 +6164,22 @@ private BoundExpression BindPointerElementAccess(ExpressionSyntax node, BoundExp
             return new BoundPointerElementAccess(node, expr, index, CheckOverflowAtRuntime, pointedAtType, hasErrors);
         }
 
+        private bool ReportRefOrOutArgument(AnalyzedArguments analyzedArguments, DiagnosticBag diagnostics)
+        {
+            int numArguments = analyzedArguments.Arguments.Count;
+            for (int i = 0; i < numArguments; i++)
+            {
+                RefKind refKind = analyzedArguments.RefKind(i);
+                if (refKind != RefKind.None)
+                {
+                    Error(diagnostics, ErrorCode.ERR_BadArgExtraRef, analyzedArguments.Argument(i).Syntax, i + 1, refKind.ToDisplayString());
+                    return true;
```

Comment:
This is the existing behavior for the contexts in which this is diagnosed.

---
In reply to: [75064582](https://github.com/dotnet/roslyn/pull/13091#discussion_r75064582) [](ancestors = 75064582)